cmd/puniversald/query.go

```diff
@@ -0,0 +1,329 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
```

Comment:
Every pchain lib reference needs to be changed since we are pointing to our github repo now